<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Davids-MBP" failures="1" tests="3" name="AuthTest" time="3.251" errors="0" timestamp="2025-01-01T22:41:14 GMT">
  <testcase classname="api.tests.AuthServiceTest" name="testSuccessfulAuthentication" time="1.499"/>
  <testcase classname="api.tests.AuthServiceTest" name="testAuthenticationWithInvalidCredentials" time="0.912"/>
  <testcase classname="api.tests.AuthServiceTest" name="testServerErrorDuringAuthentication" time="0.564">
    <failure type="java.lang.AssertionError" message="Error message should indicate server-side issue expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Error message should indicate server-side issue expected [true] but found [false]
at api.tests.AuthServiceTest.testServerErrorDuringAuthentication(AuthServiceTest.java:94)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testServerErrorDuringAuthentication -->
</testsuite> <!-- AuthTest -->
